Assets/MRTK/SDK/Experimental/Features/Utilities/StabilizationPlaneModifier.cs

```diff
@@ -310,14 +311,21 @@ private Vector3 ConfigureOverridePlaneHelper(Vector3 position, float deltaTime)
             gazeToPlane.Normalize();
             planePosition = gazeOrigin + (gazeToPlane * currentPlaneDistance);
 
-#if UNITY_WSA
+#if UNITY_2019_3_OR_NEWER
+            XRSubsystemHelpers.DisplaySubsystem?.SetFocusPlane(planePosition, OverridePlane.Normal, velocity);
```

Comment:
On 2020 and newer, I'm assuming DisplaySubsystem should be null if legacy XR is active right?

Just curious otherwise it would be interesting seeing two different calls to set the focus point/point.

Reply:
Correct, if legacy XR is active the subsystems should all not be running (if any are installed) and this helper call here will return null.